server/src/main/java/org/elasticsearch/index/fieldvisitor/CustomFieldsVisitor.java

```diff
@@ -10,21 +10,24 @@
 import org.apache.lucene.index.FieldInfo;
 
 import java.util.HashSet;
+import java.util.List;
 import java.util.Set;
 
 /**
- * A field visitor that allows to load a selection of the stored fields by exact name or by pattern.
```

Comment:
This comment is well out of date so I took the opportunity to make it a bit more relevant